scripts/play.ts

```diff
@@ -0,0 +1,550 @@
+        "https://github.com/prismicio-community/nextjs-starter-prismic-minimal-ts.git",
+        dir,
+        { dryRun: options.dryRun },
+      );
+
+      await updatePackageJSON(
+        dir,
+        { scripts: { "next:dev": "next dev --port=8001" } },
+        { dryRun: options.dryRun },
+      );
+
+      await updateSliceMachineConfig(
+        dir,
+        { localSliceSimulatorURL: "http://localhost:8001/slice-simulator" },
+        { dryRun: options.dryRun },
+      );
+
+      break;
+    }
+
+    case "nuxt": {
+      await cloneGitRepo(
+        "https://github.com/prismicio-community/nuxt-starter-prismic-minimal.git",
+        dir,
+        { dryRun: options.dryRun },
+      );
+
+      await updatePackageJSON(
+        dir,
+        { scripts: { "nuxt:dev": "nuxt dev --port=8001" } },
+        { dryRun: options.dryRun },
+      );
+
+      await updateSliceMachineConfig(
+        dir,
+        { localSliceSimulatorURL: "http://localhost:8001/slice-simulator" },
+        { dryRun: options.dryRun },
+      );
+
+      break;
+    }
+
+    case "sveltekit": {
+      await cloneGitRepo(
+        "https://github.com/prismicio-community/sveltekit-starter-prismic-minimal.git",
+        dir,
+        { dryRun: options.dryRun },
+      );
+
+      break;
+    }
+
+    default: {
+      throw new CommandError(`Unsupported framework: ${options.framework}`);
+    }
+  }
+
+  if (!options.dryRun) {
+    await fs.rm(new URL("./.git", dir), { recursive: true });
+    await fs.rm(new URL("package-lock.json", dir));
+  }
+
+  await updatePackageJSON(dir, { name }, { dryRun: options.dryRun });
+
+  // Update scripts to support the monorepo and the given stage.
+  if (options.stage === "prod") {
+    await updatePackageJSON(
+      dir,
+      { scripts: { slicemachine: START_SLICEMACHINE_SCRIPT } },
+      { dryRun: options.dryRun },
+    );
+  } else {
+    await exec("yarn", ["add", "--dev", "cross-env"], {
+      cwd: dir,
+      stdio: "inherit",
+      dryRun: options.dryRun,
+    });
+
+    await updatePackageJSON(
+      dir,
+      {
+        scripts: {
+          slicemachine: `cross-env SM_ENV=${options.stage} ${START_SLICEMACHINE_SCRIPT}`,
+        },
+      },
+      { dryRun: options.dryRun },
+    );
+  }
+
+  // Update Slice Machine configuration with the correct API endpoint (if needed).
+  if (options.stage === "staging") {
+    await updateSliceMachineConfig(
+      dir,
+      { apiEndpoint: `https://${name}.cdn.wroom.io/api/v2` },
+      { dryRun: options.dryRun },
+    );
+  } else if (options.stage === "development") {
+    const apiEndpoint = new URL(
+      "./api/v2",
+      process.env.wroom_endpoint ?? "https://cdn.wroom.io",
+    );
+    apiEndpoint.hostname = `${name}.${apiEndpoint.hostname}`;
+
+    await updateSliceMachineConfig(
+      dir,
+      { apiEndpoint: apiEndpoint.toString() },
+      { dryRun: options.dryRun },
+    );
+  }
+
+  await exec(
+    "npx",
+    [
+      "@slicemachine/init@latest",
```

Comment:
I felt it was safer to use the published known-working version rather than the development build. It's possible we could break `@slicemachine/init` locally and not be able to start a new playground.

If we want to test the local version of `@slicemachine/init`, we can call it directly somewhere with more control rather than through `yarn play`.

What do you think?

Reply:
Maybe it can be an option?
I think being able to test with the source code will allow us to use the play script in the CI and so test that the init works and that all e2e tests work after an init from the source code, like users will do, WDYT?

Reply:
With what you said, maybe we can only use a version relative to the source code and so no need for an option.

We just need to be sure the way we are doing the init with a local version is the same as a normal init so we have a real reproduction of a prod use case.

Reply:
Okay, this all works for me. 🙂 We are also using the local `start-slicemachine` script, so my argument doesn't work since we could also break that script during dev.

You can see the change here: https://github.com/prismicio/slice-machine/pull/1327/commits/ce650e660cc8e8ca5de83bf3ecb0797fa5886796